From 292d42e454e658cef2f780f12dfdf89fa32454c6 Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Wed, 28 Feb 2018 19:57:56 +0530
Subject: [PATCH 0311/1852] drm: xlnx: mixer: Adding more colors in supported
 list

commit b31969d9aa86565b31fb999dd61f5faf8ad1ba8a from
https://github.com/Xilinx/linux-xlnx.git

Adding more colors in mixer's sanity table of color list

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_mixer.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/gpu/drm/xlnx/xlnx_mixer.c b/drivers/gpu/drm/xlnx/xlnx_mixer.c
index 26e390bfd32b..e4a05c5a14a5 100644
--- a/drivers/gpu/drm/xlnx/xlnx_mixer.c
+++ b/drivers/gpu/drm/xlnx/xlnx_mixer.c
@@ -115,6 +115,13 @@ static const u32 color_table[] = {
 	DRM_FORMAT_AYUV,
 	DRM_FORMAT_NV12,
 	DRM_FORMAT_NV16,
+	DRM_FORMAT_Y8,
+	DRM_FORMAT_Y10,
+	DRM_FORMAT_XVUY2101010,
+	DRM_FORMAT_VUY888,
+	DRM_FORMAT_XVUY8888,
+	DRM_FORMAT_XV15,
+	DRM_FORMAT_XV20,
 };
 
 /*********************** Inline Functions/Macros *****************************/
-- 
2.31.1

